samwise.js

```diff
@@ -1,9 +1,8 @@
 /**
```

Comment:
SCRIPT-I like how you've used promise chaining here, it makes the code flow very clear.